kafka-agent/src/test/java/io.strimzi.kafka.agent/KafkaAgentTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.agent;
```

Comment:
This source file appears to be in a directory called `io.strimzi.kafka.agent`, rather than nested directories `io/strimzi/kafka/agent`, which would be the norm for Java.